### DISCUSSION TITLE
A way to send to all panes a command

### DISCUSSION THREAD

**Discussion by 64br137** [Eugeny/tabby#4880] [Q&A]
There's a way to send same command to all panes at the same time. Without "quick-cmds" plugin?
Since for deployment on several servers, it's a concurrent and non-easy tasks doing one command at a time (quick-command).
Something like "Multiexec" on Mobaexterm does.

There's a keyboard shortcut I can't find maybe?

**Comment by Eugeny** [comment]
Yes, you can use the "focus all panes" hotkey and then type or paste the command - `Ctrl-Shift-I` or `⌘-Shift-I`

**Reply by 64br137** [reply]
Hi Eugeny! Thanks for your answer. Great work by the way! Tried that on Windows. No success. It's a capital "i"?

**Reply by Eugeny** [reply]
Yes, it's the i key. Check Settings - Hotkeys to make sure it's the right one

**Comment by 64br137** [comment]
Thanks! Will do. Have a nice weekend.

**Comment by siu91** [comment]
it can not work on Mac, with v1.0.171
<img width="1329" alt="image" src="https://user-images.githubusercontent.com/4889213/155057717-00adc2ed-de68-4a84-87be-4f8fd40ae3c5.png">

**Reply by siu91** [reply]
@Eugeny  It is recommended to put this function in the top menu bar.

**Comment by nproctor-dev** [comment]
For anyone finding this later: Tabby does have the built-in “Focus all panes” broadcast feature Eugeny mentioned.

If you're looking for something closer to MobaXterm's persistent MultiExec workflow, including visible controls, I ended up building a plugin called tabby-multiexec.

It keeps MultiExec active while switching between panes, lets individual panes be dynamically excluded from broadcast, and adds toolbar controls for MultiExec, tiling, returning the sessions to tabs, and full-terminal clipboard copy.

Docs/screenshots:
[https://github.com/nproctor-dev/tabby-multiexec-support](https://github.com/nproctor-dev/tabby-multiexec-support?utm_source=chatgpt.com)

Currently validated with Tabby 1.0.235–1.0.236 on Linux/Kubuntu. Other platforms may work but haven't been formally validated yet.

Disclosure: I'm the plugin author. I built it after running into this same workflow while moving away from MobaXterm.
